pom.xml

```diff
@@ -354,6 +354,12 @@
                 ${surefire-log4j.file}
               </log4j.configuration>
             </systemPropertyVariables>
+           <!--properties>
+                    <property>
+                        <name>listener</name>
+                        <value>org.apache.hudi.PrintOutCurrentTestRunListener</value>
+                    </property>
+                </properties-->
```

Comment:
remove if not needed?

Reply:
I am still triaging some timeouts. patch is not yet ready for review.